update webhook doc

xtornasol512 opened the pull request:
## Description

Update webhook documentation

## Changes

- Update webhook docs
- Add Get Prescription endpoint
- Add ref to Patient Landing and Get Rx doc

xtornasol512 committed:
update webhook doc

ebarojas committed:
Update copy on webhooks.md

ebarojas reviewed (changes requested):
@xtornasol512 ¿Cómo se agrega un webhook?

Esta es la parte más importante que falta. Documentar cómo se agrega un webhook-
- como configuración de los miembres de un Hospital
- Con una campaña ATM en qué campo
- hay otro modo? especificado por médico?

xtornasol512 committed:
update: add how to create webhooks, promo_code, and faq

- Add how to create webhook
- Add how to add a webhook to new users
- Add how to create a PROMO CODE
- Add some other suggestions to create users with webhooks

xtornasol512 committed:
udpate: Update copies

- Update copy on webhook actions

xtornasol512 committed:
update: add description about medic list endpoint

- Add docs for medic limit description

xtornasol512 commented:
> @xtornasol512 ¿Cómo se agrega un webhook?
> 
> Esta es la parte más importante que falta. Documentar cómo se agrega un webhook-
> 
> * como configuración de los miembres de un Hospital
> * Con una campaña ATM en qué campo
> * hay otro modo? especificado por médico?

Nuevos cambios en la doc:

Se agrego como crear, añadir webhooks de forma convencional.
Se agrego como usar los promo code, y pedirlos antes a un admin.
Se agrego detalles del limite del endpoint de medics

ebarojas commented on docs/api/medic.md:
```diff
@@ -18,7 +18,14 @@ The following parameters may be included as part of a URL query string.
-#### GET Medics List 
+:::note
+
+The maximum limit for the MEDIC LIST endpoint is 100 records per request. If you request more than 100 items, only the first 100 records will be returned. However, this doesn’t mean you can’t retrieve all your records—pagination is available and can be used to access the entire dataset across multiple requests.
```
👍

ebarojas committed:
Update webhooks.md

ebarojas reviewed (approved):
Cool, mucho mejor.

🚀
